lib/Sema/CSApply.cpp

```diff
@@ -5399,10 +5399,15 @@ Expr *ExprRewriter::coerceCallArguments(Expr *arg, AnyFunctionType *funcType,
 
       // Otherwise, create a call of the default argument generator.
       } else {
-        defArg =
-            new (tc.Context) DefaultArgumentExpr(callee, paramIdx,
-                                                 arg->getStartLoc(),
-                                                 param.getParameterType());
+        const ParamDecl *defaultParam = getParameterAt(cast<ValueDecl>(callee.getDecl()), newArgs.size());
```

Comment:
Isn't callee.getDecl() already a ValueDecl?

Also a nitpick: we try to keep lines < 80 columns

Reply:
Yep, fixed.